mods/cnc/rules/misc.yaml

```diff
@@ -22,8 +22,18 @@ CRATE:
 		Condition: cloak-crate-collected
 	GiveMcvCrateAction:
 		SelectionShares: 0
-		NoBaseSelectionShares: 120
+		NoBaseSelectionShares: 50
 		Units: mcv
+	ExplodeCrateAction:
```

Comment:
The nuke won't damage the unit, which triggered the crate/explosion. That's unexpected.